packages/db-objects-redis/src/lib/objects/objectsInRedisClient.ts

```diff
@@ -1549,7 +1543,7 @@ export class ObjectsInRedisClient {
         return tools.maybeCallbackWithError(callback, null, result);
```

Comment:
Can't comment on the correct line. I mean this one, which exists twice in the above block:
```ts
file: dirs.shift() as string,
```
That should be
```ts
file: dirs.shift()!,
```
instead, since all you want to do is remove the `| undefined` from the type and not overwrite it completely.

Reply:
thanks did not knew this operator until now